cmd/kogito/command/test/common.go

```diff
@@ -33,6 +35,14 @@ var (
 	rootCommand *cobra.Command
 )
 
+const (
+	// Test constants for config file, based on context ones
+	DefaultConfigPath = context.DefaultConfigPath
```

Comment:
We really need to export those `const`s? If not, make them private please or add a comment to it so the lint tool won't make noise. Run the `make test` command in the root directory and observe the lint fails for more info.

Reply:
updated